packages/bus-query-operation/package.json

```diff
@@ -41,7 +41,8 @@
     "@comunica/core": "^1.0.0"
   },
   "devDependencies": {
-    "@comunica/core": "^1.19.2"
+    "@comunica/core": "^1.19.2",
+    "@comunica/types": "^1.19.2"
```

Comment:
This would actually have to be an actual dependency. Also for all other packages (that use these types) for that matter. Strange that the linter didn't trip over this, as I thought I defined strict linter rules for this...

Reply:
The general issue of errors not being 100% deterministic across environments is an annoying aspect of TS and TS linting, at least insofar as my limited experience goes. Will do.

Reply:
Done!